.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+*	@percona/everest-backend-devs
+/ui	@percona/everest-frontend-devs
```

Comment:
how will it work? I believe the FE will be notified only if there are the changes in the /ui folder. Will the BE devs be notified for all the FE changes? If so, is there a way to avoid this?

Reply:
```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2

# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global
# owner(s) will be requested for a review.
*.js    @js-owner #This is an inline comment.
```

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file

Reply:
Sorry I should have added this info to the PR description.

Reply:
Thanks for the explanation!